app/scripts/constants.js

```diff
@@ -52,6 +52,10 @@ window.OPENSHIFT_CONSTANTS = {
   // true indicates that deployment metrics should be disabled on the web console overview
   DISABLE_OVERVIEW_METRICS: false,
 
+  // true indicates that none of the routers support wildcard subdomains and
+  // removes the option from the route creation form.
+  DISABLE_WILDCARD_SUBDOMAINS: false,
```

Comment:
> I vote that we keep it false for now.

Note that the setting disables web console support for wildcards. So leaving it false will *enable* the feature. Just confirming that's what you mean :)